Makefile

```diff
@@ -0,0 +1,17 @@
+IDIR = include

```

Comment:
We're currently in a transition phase and use both, Make and gyp, so we'll need both. But we want to get to only gyp eventually.
Also note that the build system changes you'll have to make to NSS to make this work are minimal. Don't invest too much time into it.